_site/components/types.js

```diff
@@ -1,6 +1,6 @@
 /**
  * @typedef {object} Config
- * @property {'avif' | 'webply' | 'png8' | 'pjpg'} format
+ * @property {'avif' | 'jxl' | 'webply' | 'png8' | 'pjpg'} format
```

Comment:
Sorry to arrive late to this… The [docs](https://www.fastly.com/documentation/reference/io/format/) aren’t super-clear, so while I don’t think `jxl` means lossless (although, for some reasons they support lossless WebP), I think we need `pjxl`…

Reply:
You are right, I need to fix this

Reply:
https://github.com/guardian/image-comparison-tool/pull/11

Reply:
I will test in some Thorium thing. It’s ridiculous we need to advocate for browsers [to do things properly](https://bugs.webkit.org/show_bug.cgi?id=272350)… Amateurs!